deploy-blueprint/library/util/playbooks/modakconfig.yml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: all
+  become: yes
+
+  tasks:
+    - name: Generating config for MODAK API
+      template:
+        src: "iac-model.ini.j2"
+        dest: "/tmp/iac-model.ini"
+        mode: "0755"
```

Comment:
Where are we deploying this exactly? Just feeling a bit uncomfortable to have a file with credentials and with world-readable permissions in `/tmp`. And "blueprints" suggests this might used in other environments as well.

Reply:
This only deploys on infrastructure we control. Unfortunately, MODAK doesn't have any mechanism for getting secrets that isn't just reading this file. It's definitely less than ideal, but the whole database is currently checked in to the repository and there isn't any secret information that goes into it. I don't think it's worth the effort to secure this when it doesn't protect any private information.